code/script.js

```diff
@@ -1,9 +1,29 @@
 // A function that adds and remove the class "active" on the section you click on.
 // We haven't really talked about ´this´ yet, but we will... ;)
 // console.log(this) to see how it works
-function toggle() {
-  this.classList.toggle("active")
-}
+//function toggle() {
+//this.classList.toggle("active")
+//}
```

Comment:
Code that we don't use can be deleted before making the pull request to "clean up" the code a bit! If you want to save the code you can add the code to a commit and then remove it. Then you can just go back to that commit when you want to see the code again